File: works/1712.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split(' ');
// let input = fs.readFileSync('input.txt').toString().split(' ');

A = parseInt(input[0])
B = parseInt(input[1])
C = parseInt(input[2])
benefit = C- B 
if(benefit <= 0) console.log(-1) 
else console.log(Math.floor(A / benefit) + 1)

File: works/1712.dart
import 'dart:io';

void main()
{
    var input = stdin.readLineSync()!.split(' ');
    int A = int.parse(input[0]);
    int B = int.parse(input[1]);
    int C = int.parse(input[2]);
    int benefit = C - B;
    if(benefit <= 0) print(-1);
    else print((A / benefit).floor() + 1);


}